packages/create-pantheon-decoupled-kit/src/templates/tailwind-shared/postcss.config.cjs.hbs

```diff
@@ -0,0 +1,16 @@
+module.exports = {
+	plugins: {
+		tailwindcss:{},
+		autoprefixer:{},
+		{{#if convertCssModules}}
+		'@csstools/postcss-global-data':
+		{ 
+		files: [
+			'./styles/globals.css',
+		],
+		},
+		'postcss-custom-properties':{ preserve: false },
```

Comment:
Can these options be supplied via the postcss CLI when it is called instead? Thinking about how to keep all the deps for this out of the final output.
If necessary you may need to write some TS that uses these packages directly to do the transform and rewrite it to the file system, like we are doing in `addWithDiff`.